api/tests/test_asset_log.py

```diff
@@ -5,6 +5,7 @@
 from django.contrib.auth import get_user_model
 from django.core.exceptions import ValidationError
 from rest_framework.test import APIClient
+from datetime import datetime
```

Comment:
It's standard practice to import datetime as dt. You could do this `from datetime import datetime as dt`.

Reply:
wouldn't call it standard practice.. i'd say it applies when you're importing other objects from the datetime module